docs/contribute/dev-setup.md

```diff
@@ -0,0 +1,122 @@
+Thank you for considering to contribute to this project. This page shows you how to setup your local
+development environment.
+
+### [Optional] Install Docker
```

Comment:
Nittiest of nit picks! These aren't optional steps; I'd rename to "pre-requisites" or "system requirements"  and still include pointers on installing etc.

Reply:
Good point. I'll change this